src/Particles/BaseParticle.h

```diff
@@ -147,6 +147,9 @@ class BaseParticle {
 
 	/// Needed for specific base pairing
 	int btype;
+	
+	/// Needed for DNA-RNA interactions
+	char acid_type;
```

Comment:
@eryykr Can we do this bookkeeping in the DRH interaction rather than here? Do we foresee any issues?

Reply:
I think we could so long as there is a way of distinguishing between DNA and RNA nucleotides without acid_type

Reply:
OK! Have a look at my changes in https://github.com/lorenzo-rovigatti/oxDNA/pull/72/commits/b69562396d05e0c374e383b9f19d834799e02b68 and let me know if you agree with them.

Reply:
Looks good!